website/pages/en/get-involved.js

```diff
@@ -34,8 +34,7 @@ function Help(props) {
     },
     {
       title: `Are you using FDC3?`,
-      content: `If you an existing individual or corporate user of the FDC3 standard, we would love to hear from you: just email the [FDC3 General List](fdc3@finos.org) with details about how you are using the standard. If you'd like to be listed as a standard user in our [homepage](fdc3.finos.org), you can directly [send a pull request to update our website](https://github.com/finos/fdc3/website/data/users.json) or, if listing your logo publicly requires legal evaluation, you can reach out privately to the [FDC3 Product Management Committee](fdc3-private@finos.org).`,
-    },
```

Comment:
@aitana16 I think you inadvertently removed a `{` character during this change, which is making the build fail...

Reply:
thank you very much @rikoe, I fixed it and it looks like the build is passing now